src/Microsoft.ML.Featurizers/Common.cs

```diff
@@ -89,40 +99,36 @@ protected override bool ReleaseHandle()
     internal class ErrorInfoStringSafeHandle : SafeHandleZeroOrMinusOneIsInvalid
     {
         [DllImport("Featurizers", EntryPoint = "DestroyErrorInfoString", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
-        private static extern bool DestroyErrorInfoString(IntPtr errorString, IntPtr errorStringSize);
+        private static extern bool DestroyErrorInfoString(IntPtr errorString);
 
```

Comment:
Can you please explain why the size is no longer necessary?

Reply:
We have changed all strings to be c strings with the final character as null. This made interop of other types easier (like vectors of string). So now instead of getting the size back we just scan until the null character.